### PR TITLE
disable neon on macOS aarch64

### PR DESCRIPTION
**Pull request by dpzmick**
See:
- https://gitlab.freedesktop.org/pixman/pixman/-/issues/59
- https://gitlab.freedesktop.org/pixman/pixman/-/issues/69
- https://github.com/Homebrew/homebrew-core/blob/e00896f32d894cca0204df6bf3dbbecc49f94a3c/Formula/pixman.rb

### CODE REVIEW

File: var/spack/repos/builtin/packages/pixman/package.py [hunk 1]
```diff
@@ -62,6 +62,8 @@ def configure_args(self):
         args = ["--enable-libpng", "--disable-gtk"]
 
         if sys.platform == "darwin":
+            # neither style of acceleration work on macOS
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/pixman/package.py:65]: The change LGTM, but can you put the links in the PR description in the comment above?

**Review by alalazo — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/pixman/package.py (hunk 1 above)
Comment (dpzmick) [reply var/spack/repos/builtin/packages/pixman/package.py:65]: done!

**Comment by alecbcs** [comment]
@alalazo

**Comment by alalazo** [comment]
From my side this just needs the conflict to be resolved, and then it's :heavy_check_mark:

**Comment by alalazo** [comment]
Superseded by #36982

**Pull request closed without merging**